.github/dependabot.yml

```diff
@@ -1,8 +1,7 @@
 version: 2
 updates:
-  - package-ecosystem: "mix"
-    directory: "/"
-    target-branch: "main"
-    insecure-external-code-execution: allow
```

Comment:
I think you need this `insecure-external-code-execution: allow`, but let's try without it.